Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node
+
```

Comment:
We may want to add `"&& rm /home/AMO-Tools-Suite/CMakeCache.txt` after ln 6. I received an obtuse error because the file still existed in proj directory from old builds.

Reply:
from the same root directory (where the docker-compose.yml file is) you can either run the code in container or build it your self manually, if you build locally on your machine, make sure you clean your root directory and remove all your local build files and folders.

Reply:
For sure cmake cache should not be committed, depending on the tools / compilers used it makes lot of fluff, most of them are excluded in our current git  ignore.

for individual  machine clean up, check the git repo any folder / files, any thing not in it currently can be deleted. Ofcource I dont mean delete the new code files you create yourself, but just the ones you see in root folder from build process.

Reply:
For the reason of choice of usage, I wont create any cleanup scripts or prefer anyone doing it that goes in repo, individuals can create for themselves if needed.